am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/DefaultRoleSetupDataFactory.java

```diff
@@ -36,6 +38,23 @@ private DefaultRoleSetupDataFactory() {
         return attributePermission;
     }
 
+    public static Map<JsonPointer, Pair<Set<Permission>, SecurityClassification>> grantMultipleResources(){
```

Comment:
Are you granting anything within that method? Rename to start with `create...`.

Reply:
Since it is not reusable at this point please move that method to integration tests itself. It will make test more readable because data setup and assertions will be kept together.

Reply:
👍

Reply:
👍